### ISSUE TITLE
Swag Opportunities

### ISSUE THREAD

**Issue by StrangeWill** [devanooga/devanooga-meta#57]
Lots of interest for Devanooga branded Orange shirts (tri-blend?)

Tumblers, mugs? Need to run a poll, get some ideas together.

**Comment by thetest3r** [comment]
T shirts are cool. 
Polos would allow me to wear devanooga to work xD.
Depends on the design of the mug.

Magnets are a little less invasive than stickers, so that wouldn't be a terrible idea.

Running up preorders and collecting the money first would definitely offset any risk.

**Comment by StrangeWill** [comment]
Yeah, I mean even if we don't do preorders but I have a solid idea of what people are interested in we can fund this up-front, the main thing that preorders gets me is a solid idea of _what_ to order and how much, but I'm not against keeping some inventory around (we'll have to anyway just by the nature of mass producing this stuff).

I also got some connections that may help us get this stuff at pretty bottom dollar -- I just need to finish and send out our questionnaire covering these items.

The car decals would likely be special order, but the places I've seen will allow us to do that.

**Comment by StrangeWill** [comment]
https://docs.google.com/forms/d/e/1FAIpQLSdRMVZKvMNTFe8PFn1MsSJ_1syAjeJEYIKbCkVd1zjgPg4duA/viewform?usp=sf_link

We've had this up, need to get stuff together to get this done.

**Comment by thetest3r** [comment]
Got this: https://devanooga-brand-identity-guide.netlify.app/

And a rough mockup/approved design looking like:
https://devanooga.slack.com/archives/C4E4UNSKC/p1588332616017000
